SparkPost/2web2ui
# SA-1079: Add sending-ip to all messages page

### What Changed
 - Renamed action and function names to be singular version of filter. 
IE: mailbox_providers->mailbox_provider & regions-> region & test->tests.
 - Added action/reducer for getting tests grouped by sending-ip.
 - Added sending-ip as an option for the all messages page. 

### How To Test
 - on prod account 108, go to `/inbox-placement/details/76/sending-ip/147.253.215.24` and verify that everything is working.
- on appteam staging, you can use `/inbox-placement/details/49/sending-ip/209.85.166.66`
